src/main/java/core/basesyntax/service/operations/OperationDecreaseHandler.java

```diff
@@ -0,0 +1,27 @@
+package core.basesyntax.service.operations;
+
+import core.basesyntax.dao.FruitDao;
+import core.basesyntax.exeptions.InvalidQuantityException;
+import core.basesyntax.model.Fruit;
+import java.util.Optional;
+
+public class OperationDecreaseHandler implements OperationHandler {
+    private static final String ERROR_MESSAGE
```

Comment:
It's ok to write an error message directly when instantiating the exception object.

Reply:
Done!